spec/export/index.md

```diff
@@ -299,36 +316,36 @@ Content-Type: application/json
 Required Fields:
```

Comment:
The aim from https://github.com/HL7/bulk-data/issues/21 is that these JSON fields will be documented in their own table, as https://cds-hooks.org/specification/1.0/

![image](https://user-images.githubusercontent.com/313089/61807813-e0207500-adff-11e9-8b01-a2c66c07f222.png)

Reply:
@jmandel gotcha, `result body` i was thinking req params.  https://github.com/HL7/bulk-data/pull/34/commits/fd7f9d4c7d76a542fa570913ccc3b68a2fae8881